src/UsesResolver.php

```diff
@@ -0,0 +1,134 @@
+use PhpParser\Node\Stmt\Use_;
+use PhpParser\Node\Stmt\UseUse;
+use PhpParser\ParserFactory;
+
+class UsesResolver
+{
+    /**
+     * @var array
+     */
+    protected $classAliases = [];
+
+    /**
+     * @return array
+     */
+    public function getClassAliases()
+    {
+        return $this->classAliases;
+    }
+
+    /**
+     * @param string $classFQN
+     * @return $this
+     */
+    public function loadFromClass(string $classFQN)
+    {
+        return $this->loadFromFile(
+            $classFQN,
+            (new \ReflectionClass($classFQN))->getFileName()
+        );
+    }
+
+    /**
+     * @param string $classFQN
+     * @param string $filename
+     * @return $this
+     */
+    public function loadFromFile(string $classFQN, string $filename)
+    {
+        return $this->loadFromCode(
+            $classFQN,
+            file_get_contents(
+                $filename
+            )
+        );
+    }
+
+    /**
+     * @param string $classFQN
+     * @param string $code
+     * @return $this
+     */
+    public function loadFromCode(string $classFQN, string $code)
+    {
+        $classFQN = ltrim($classFQN, '\\');
+
+        $namespace = rtrim(
+            preg_replace(
+                '/([^\\\\]+)$/',
+                '',
+                $classFQN
+            ),
+            '\\'
+        );
+
+        $parser = (new ParserFactory())->create(ParserFactory::PREFER_PHP7);
+
+        /** @var Namespace_ $namespaceData */
+        $namespaceData = Arr::first(
+            $parser->parse($code),
+            function ($node) use ($namespace) {
+                return $node instanceof Namespace_
+                    && $node->name->toCodeString() === $namespace;
+            }
+        );
+
+        foreach ($namespaceData->stmts as $stmt) {
+            if ($stmt instanceof Use_) {
+                if ($stmt->type !== Use_::TYPE_NORMAL) {
+                    continue;
+                }
+
+                foreach ($stmt->uses as $use) {
+                    /** @var UseUse $use */
+
+                    $this->addClassAlias(
+                        '\\' . $use->name->toCodeString(),
+                        $use->alias ? $use->alias->name : null
+                    );
+                }
+            } elseif ($stmt instanceof GroupUse) {
+                foreach ($stmt->uses as $use) {
+                    /** @var UseUse $use */
+
+                    $this->addClassAlias(
+                        '\\' . $stmt->prefix->toCodeString() . '\\' . $use->name->toCodeString(),
+                        $use->alias ? $use->alias->name : null
+                    );
+                }
+            }
+        }
+
+        return $this;
+    }
+
+    /**
+     * @param string $classFQN
+     * @param string|null $alias
+     * @return void
+     */
+    protected function addClassAlias(string $classFQN, string $alias = null)
+    {
+        if ($alias === null) {
+            $alias = class_basename($classFQN);
```

Comment:
Unfortunately can't use global functions from Support/helpers.php in a library.

ide-helper, whilst Laravel specific, may be used just with its individual illuminate/* packages and must not depend on such functions.

Fix in this case: sorry, just copypaste the code over 😬